src/mysql/types.cr

```diff
@@ -315,4 +322,28 @@ abstract struct MySql::Type
     end
   end
   decl_type Geometry, 0xffu8
+
+  # Parse the JSON column type
+  decl_type Json, 245_u8, JSON::Any do
+    def self.write(packet, v : String)
+      packet.write_lenenc_string v
+    end
+
+    def self.write(packet, v : ::JSON::Any)
+      packet.write_lenenc_string v.to_json
+    end
+
+    def self.read(packet)
+      str = packet.read_lenenc_string.lchop("\u0013")
```

Comment:
Does MySQL sends trailing `\u0013` on all json values? Is there a documentation/reference for this?